ee/vellum_ee/workflows/display/tests/workflow_serialization/generic_nodes/test_adornments_serialization.py

```diff
@@ -216,6 +216,27 @@ def test_serialize_node__try(serialize_node):
     )
 
 
+def test_serialize_node__try__no_display():  # GIVEN an adornment node
```

Comment:
[nit] for both the retry one and this one, let's have the comment in the body of the method haha